pom.xml

```diff
@@ -89,7 +89,7 @@
     </scm>
 
     <properties>
-        <revision>1.0.0</revision>
+        <revision>1.0.1</revision>
```

Comment:
search CommonVersion class & also need update it 

btw, we need a better way to modify the num, at least add a comment in pom?

Reply:
> If it is packaged in a jar, we cannot get the version from the pom.xml, so it fallback to the default version.

OK, we could consider these way: (way2 is the easiest)
1. check if somewhere possible for us to put the `version` into the `jar` file (like `meta` file/info of a jar?) 
2. just leave a comment before `revision` param in pom file (devs could see the `NOTE`)
3. remove the common version check, seems it's not necessary to check it?

Reply:
> 

also need to update RpcVersion.java

Reply:
let's keep the status quo until we find a perfect way